benchmark_runner/common/template_operations/templates/hammerdb/internal_data/hammerdb_vm_template.yaml

```diff
@@ -100,10 +100,14 @@ spec:
         cores: {{ cores }}
         threads: 1
         image: quay.io/cloud-bulldozer/{{ db_image }}
+        {%- if limits_memory %}
```

Comment:
@RobertKrawitz, I dont think we need those conditions because always we are using the limit, its also relate all the conditions below

Reply:
Having these conditions gives us flexibility later if we want to remove them.

Reply:
For now we are going to keep the limits for all so I dont think we need it. it just makes the templates less readable, do you agree with it ?

Reply:
Hard to argue that it makes it harder to read.  Updated, but I'd like not to squash this out in case we want to do it later.